src/content/docs/tips/biome.mdx

```diff
@@ -0,0 +1,138 @@
+        ```
+      </TabItem>
+      <TabItem label="pnpm" icon="pnpm">
+        ```sh
+        pnpm add --save-dev --save-exact @biomejs/biome
+        ```
+      </TabItem>
+      <TabItem label="yarn" icon="seti:yarn">
+        ```sh
+        yarn add --dev --exact @biomejs/biome
+        ```
+      </TabItem>
+      <TabItem label="bun" icon="bun">
+        ```sh
+        bun add --dev --exact @biomejs/biome
+        ```
+      </TabItem>
+    </Tabs>
+
+2. Next create the biome configuration:
+
+    <Tabs syncKey="biome">
+      <TabItem label="npm" icon="seti:npm">
+        ```sh
+        npx @biomejs/biome init
+        ```
+      </TabItem>
+      <TabItem label="pnpm" icon="pnpm">
+        ```sh
+        pnpm biome init
+        ```
+      </TabItem>
+      <TabItem label="yarn" icon="seti:yarn">
+        ```sh
+        yarn biome init
+        ```
+      </TabItem>
+      <TabItem label="bun" icon="bun">
+        ```sh
+        bunx biome init
+        ```
+      </TabItem>
+      <TabItem label="deno" icon="deno">
+        ```sh
+        deno run -A npm:@biomejs/biome init
+        ```
+      </TabItem>
+    </Tabs>
+
+3. Provide the overrides property inside biome config to support `.astro` files:
+
+    ```js
+    {
+      "overrides": [
+        {
+          "include": ["*.astro"],
+          "linter": {
+            "rules": {
+              "style": {
+                "useConst": "off",
+                "useImportType": "off"
+              }
+            }
+           }
+        }
+      ]
+    }
+    ```
+
+4. Run Biome's CLI to lint & format your project's files:
+  
+    <Tabs syncKey="biome">
+        <TabItem label="npm" icon="seti:npm">
+          ```sh
+          npx @biomejs/biome check --write .
+          ```
+        </TabItem>
+        <TabItem label="pnpm" icon="pnpm">
+          ```sh
+          pnpm biome check --write .
+          ```
+        </TabItem>
+        <TabItem label="yarn" icon="seti:yarn">
+          ```sh
+          yarn biome check --write .
+          ```
+        </TabItem>
+        <TabItem label="bun" icon="bun">
+          ```sh
+          bunx biome check --write .
+          ```
+        </TabItem>
+        <TabItem label="deno" icon="deno">
+          ```sh
+          deno run -A npm:@biomejs/biome check --write .
+          ```
+        </TabItem>
+      </Tabs>
+</Steps>
+
+<Aside>
+In case of an existing configuration for ESLint and Prettier, you might want to read Biome's guide for [Migrate from ESLint and Prettier](<https://biomejs.dev/guides/migrate-eslint-prettier/>)
+</Aside>
+
+
+## Adding an npm script
+
+You can use Biome CLI with your preferred package manager by adding following scripts in `package.json` of your project's directory:
+
+```json title="package.json" ins={3-5}
+{
+  "scripts": {
+    "format": "biome format --write .",
+    "lint": "biome lint --write .",
+    "lint:fix": "biome check --write ." // lints, formats and organizes imports
```

Comment:
Do we need to have all three options?

Reply:
also worth checking if the argument changed (I think there's --apply or something, feel free to give a docs link)

Reply:
I assume `format` is redundant considering `check` exists, I will trim that out. docs tell `apply` is [deperecated](https://biomejs.dev/reference/cli/#biome-check:~:text=%2D%2Dapply%20%E2%80%94%20Alias%20for%20%2D%2Dwrite%2C%20writes%20safe%20fixes%2C%20formatting%20and%20import%20sorting%20(deprecated%2C%20use%20%2D%2Dwrite)) to `write` 😅.